README.md

```diff
@@ -1,14 +1,36 @@
-﻿# Шаблон .Net RabbitMQ consumer сервиса
+﻿# .NET RabbitMQ consumer service template
 
-## Установка шаблона из nuget.org
-`dotnet new --install Byndyusoft.Net.RabbitMqConsumerTemplate`
+## What includes?
 
-В списке должен появиться шаблон с именем bsrabbitconsumer
+Template includes:
+- RabbitMQ package - [Byndyusoft.Messaging.RabbitMq](https://github.com/Byndyusoft/Byndyusoft.Net.RabbitMq).
+- Package for masking sensitive data in logs and traces - [Byndyusoft.MaskedSerialization](https://github.com/Byndyusoft/Byndyusoft.MaskedSerialization).
+- Health checks using method /healthz.
+- OpenTelemetry Tracing.
 
-## Создание сервиса из шаблона
-`dotnet new bsrabbitconsumer -n {Название сервиса}` - создание в текущей директории
+Projects:
+- src
+  - RabbitMqConsumerTemplate - Worker-consumer that handled incoming messages
+  - Contracts - Consumer integration contracts
 
-`dotnet new bsrabbitconsumer -o {Название сервиса}` - создание директории с сервисом
+- tests 
+  - UnitTests - Unit Tests
+  - IntegrationTests - Consumer Integration Tests
+
+## Install
+``` shell
+dotnet new --install Byndyusoft.Net.RabbitMqConsumerTemplate
+```
+
+Template with name bsrabbitconsumer should be appear in your projects list
```

Comment:
Template with [srabbitconsumer name] should [appear] in your [project] list

Reply:
++